holoviews/plotting/mpl/chart.py

```diff
@@ -61,6 +63,14 @@ def get_data(self, element, ranges, style):
         ys = element.dimension_values(1)
         return (xs, ys), style, {}
 
+    def init_artists(self, ax, plot_args, plot_kwargs):
+        xs, ys = plot_args
+        if xs.dtype.kind == 'M':
```

Comment:
Might be worth adding a comment mentioning what the 'M' code means. I assume it is the numpy datetime64 type?